zuul.d/jobs.yaml

```diff
@@ -270,6 +270,14 @@
     vars:
       ansible_test_python: 3.8
 
+- job:
```

Comment:
Should we merge a PR with just this change, before we add the 3.9 unit jobs for the other platforms?